wordpress-mobile/WordPress-Android
# Issue/9452 domain suggestions

Fixes #9452 

This PR adds a domain suggestion screen.
The base of this PR was laid out by Usama Hamid. I did some refactoring and added a couple of changes here and there.

Note: For some reason, the last suggestion in response is displayed as the first in a list. I will address this later on Flux-C side.

[![Image from Gyazo](https://i.gyazo.com/bdb846e333100e334cf1b359850acf11.png)](https://gyazo.com/bdb846e333100e334cf1b359850acf11)

[![Image from Gyazo](https://i.gyazo.com/7b71d1396f2120984dd77980f16e86e7.png)](https://gyazo.com/7b71d1396f2120984dd77980f16e86e7)

To test:

- Install the wasabi build.
- Switch to a site on a business plan with unclaimed domain credit (let me know if you need help with this).
- Navigate to Plugins screen, and from there to details of any plugin (like Woocommerce).
- Tap on Install button.
- In the dialog tap on "REGISTER DOMAIN".
- Make sure domain suggestions screen behaves as expected.

Update release notes:
